windows/src/desktop/history.md

```diff
@@ -3,11 +3,12 @@
 ## 10.0 alpha
 * Keyman Desktop moved to open source (#121)
 * Support for custom BCP-47 language codes: you can now associate a keyboard with any valid language code in Windows 8 and later
-* Keyman API extensively rewritten and improved
+* Keyman API: extensively rewritten and improved consistency
```

Comment:
"... with improved consistency."  The grammar originally here (with the 'and') is very inconsistent.